services/QuillLMS/client/app/bundles/Teacher/components/accounts/shared/agreements.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react'
+
+const Agreements = () => (
+  <p className="agreements">By signing up, you agree to our <a href="/tos">Terms of Service</a> and <a href="/privacy">Privacy&nbsp;Policy.</a></p>
+)
```

Comment:
I'm really glad you broke this out into a separate component rather than just duplicating.  Makes a lot of sense.